client/src/utils/createApiRequest.js

```diff
@@ -0,0 +1,11 @@
+export default function createApiRequest(url, method, data) {
+  fetch(url, {
+    method,
+    body: data ? JSON.stringify(data) : null
```

Comment:
Oh yeah, good call. As far as I know it doesn't short circuit, but we could totally do it that way. Just want to make sure null is set if no data is being sent.